sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -166,9 +166,9 @@ class SQLContext(@transient val sparkContext: SparkContext)
 
   protected[sql] def parseSql(sql: String): LogicalPlan = ddlParser.parse(sql, false)
 
-  protected[sql] def executeSql(sql: String): this.QueryExecution = executePlan(parseSql(sql))
+  protected[sql] def executeSql(sql: String): org.apache.spark.sql.QueryExecution = executePlan(parseSql(sql))
```

Comment:
Can we move the `o.a.s.s.QueryExecution` into another file(or a new file), and import it by a renamed the class name.